### PR TITLE
Update endpoints.py customer_payments 

### LINKED ISSUE #57: customer_payments doesn't have POST method
I was looking through the source code and see that there is no POST method for customer_payments. I was wondering if there is a way to do this manually?

### PR DESCRIPTION
**Pull request by HHry0918**
Fixes #57

Update customer_payments method to include GET, POST and DELETE (PUT is not supported according to the documentation api)

### CODE REVIEW

File: myob/endpoints.py [hunk 1]
```diff
@@ -31,7 +31,10 @@
     'Sale/CustomerPayment/': {
         'name': 'customer_payments',
         'methods': [
-            (ALL, '', 'sale customer payment type')
+            (ALL,    '', 'sale customer payment type'),
```
Comment (jarekwg) [inline myob/endpoints.py:34]: Do you mind fixing up this line as well while we're here?
```suggestion
            (ALL,    '', 'sale customer payment'),
```

**Review by jarekwg — CHANGES_REQUESTED** [review]
Hey @HHry0918,

Thanks for contributing. Could you please also update the tests accordingly?
Cheers,

File: myob/endpoints.py (hunk 1 above)
Comment (HHry0918) [reply myob/endpoints.py]: Done!

**Comment by jarekwg** [comment]
Click into github checks and update them so that they're not failing.
In your cloned repo, you can run `python setup.py test` from your root directory to see if the tests are all passing.

**Comment by HHry0918** [comment]
Hi, not sure what this error is in your test

```
Traceback (most recent call last):
  File "c:\users\harry\documents\github\pymyob_fork\tests\test_endpoints.py", line 203, in test_customer_payments
    self.assertEqual(repr(self.companyfile.customer_payments), (
AssertionError: 'Sale[31 chars]     all() - Return all sale customer payments[199 chars]ent.' != 'Sale[31 chars]                all() - Return all sale custom[250 chars]t.\n'
  Sale_CustomerPaymentManager:
-           all() - Return all sale customer payments for an AccountRight company file.
+                      all() - Return all sale customer payment types for an AccountRight company file.
? +++++++++++                                                  +++++
-     delete(uid) - Delete selected sale customer payment.
+                delete(uid) - Delete selected sale customer payment.
? +++++++++++
+                 post(data) - Create new sale customer payment.
-        get(uid) - Return selected sale customer payment.
+                   get(uid) - Return selected sale customer payment.
? +++++++++++
-      post(data) - Create new sale customer payment.
```

**Comment by jarekwg** [comment]
Take a look at the tests in this file: https://github.com/uptick/pymyob/blob/master/tests/test_endpoints.py
They need to be updated to cover the endpoints you've added. The error shows you where the test's expectation doesn't match what's getting returned.

**Comment by HHry0918** [comment]
Hi, following up on the error about test_endpoints.py.

`Traceback (most recent call last):
  File "e:\github\pymyob\tests\test_endpoints.py", line 211, in test_customer_payments
    self.assertEndpointReached(self.companyfile.customer_payments.delete, {'uid': UID}, 'DELETE', f'/{CID}/Sale/CustomerPayment/')
  File "E:\Program\anaconda3\envs\dimo\lib\unittest\mock.py", line 1337, in patched
    return func(*newargs, **newkeywargs)
  File "e:\github\pymyob\tests\test_endpoints.py", line 48, in assertEndpointReached
    mock_request.assert_called_once_with(
  File "E:\Program\anaconda3\envs\dimo\lib\unittest\mock.py", line 919, in assert_called_once_with
    return self.assert_called_with(*args, **kwargs)
  File "E:\Program\anaconda3\envs\dimo\lib\unittest\mock.py", line 907, in assert_called_with
    raise AssertionError(_error_message()) from cause
AssertionError: expected call not found.
Expected: request('DELETE', 'https://api.myob.com/accountright/DummyCompanyId/Sale/CustomerPayment/', headers={'Authorization': 'Bearer None', 'x-myobapi-cftoken': '!encoded-userpass=', 'x-myobapi-key': 'KeyToTheKingdom', 'x-myobapi-version': 'v2'}, params={}, timeout=None)
Actual: request('DELETE', 'https://api.myob.com/accountright/DummyCompanyId/Sale/CustomerPayment/DummyResourceUid/', timeout=None, headers={'Authorization': 'Bearer None', 'x-myobapi-cftoken': '!encoded-userpass=', 'x-myobapi-key': 'KeyToTheKingdom', 'x-myobapi-version': 'v2'}, params={})`

Not sure what is causing this error, I have followed the other test cases

**Comment by jarekwg** [comment]
Done in #64.
Thanks @HHry0918!

**Pull request closed without merging**